test/functional/cmdline_options_testresources/src_90/com/ibm/j9/getcallerclass/RefectionMHTests.java

```diff
@@ -108,7 +108,9 @@ public static boolean test_getCallerClass_Helper_MethodHandle_fromBootExtWithAnn
 			methodHandle = lookup.findStatic(GetCallerClassTests.class, "test_getCallerClass_MethodHandle", methodType);
 			cls = (Class<?>) methodHandle.invoke();
 
-			if ("java.lang.invoke.SecurityFrame" == cls.getName()) {
+			if (("java.lang.invoke.SecurityFrame" == cls.getName())
+			|| cls.getName().startsWith(RefectionMHTests.class.getName() + "$$InjectedInvoker/")
```

Comment:
nitpick: if this follows c coding standard, a tab is required before `||`.
Otherwise, this looks good.

Reply:
did you mean java standard? I did notice that our java test code having the extra tab and not using separate line for the closing `) {`
I also saw some code that is not using the extra tab, just not sure which format to follow

Reply:
I referred the C version though it was internal VM doc.

Reply:
fixed